src/sentry/ingest/ingest_consumer.py

```diff
@@ -24,7 +24,7 @@
 from sentry.attachments import CachedAttachment, attachment_cache
 from sentry.ingest.types import ConsumerType
 from sentry.ingest.userreport import Conflict, save_userreport
-from sentry.event_manager import save_attachment, save_transaction_events
+from sentry.event_manager import save_attachment
```

Comment:
Can you also remove `save_transaction_events`, then?

Reply:
This is used inside of event_manager and is basically the "separate codepath" I mentioned in the PR description. I *think* we would want to keep this even though maybe no longer batching, not going through the regular event manager save still sounds nice to have.